bee-common/bee-packable/src/unpacker/slice.rs

```diff
@@ -13,12 +13,13 @@ impl<'u> SliceUnpacker<'u> {
     }
 }
 
-/// Error type to be raised when [`SliceUnpacker`] does not have enough bytes to unpack something.
+/// Error type to be raised when [`SliceUnpacker`] does not have enough bytes to unpack something
+/// or when ['crate::packer::SlicePacker'] does not have enoug space to pack something.
```

Comment:
It's better to have `['SlicePacker'](crate::packer::SlicePacker)`, that way only SlicePacker shows in the docs